### PR TITLE
test(login): Verify that Google login link is correct

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

We accidentally made it impossible to log in with social auth for a couple hours, all because of… a missing trailing slash in a `href`. This shouldn't be an easy bug to ship.

## Changes

Adds a lightweight Cypress test to verify that the correct link is used, to allow the #14642 revert revert to go in safely.
This is far from end-to-end, but it's better than no check at all.

### CODE REVIEW

File: bin/e2e-test-runner [hunk 1]
```diff
@@ -92,8 +92,12 @@ setupDev() {
   python manage.py setup_dev &
 }
 
-nc -z localhost 9092 || ( echo -e "\033[0;31mKafka isn't running. Please run\n\tdocker compose -f docker-compose.dev.yml up zookeeper kafka clickhouse db redis\nI'll wait while you do that.\033[0m" ; bin/check_kafka_clickhouse_up )
-wget -nv -t1 --spider 'http://localhost:8123/' || ( echo -e "\033[0;31mClickhouse isn't running. Please run\n\tdocker compose -f docker-compose.dev.yml up zookeeper kafka clickhouse db redis.\nI'll wait while you do that.\033[0m" ; bin/check_kafka_clickhouse_up )
+service_warning() {
+  echo -e "\033[0;31m$1 isn't ready. You can run the stack with:\ndocker compose -f docker-compose.dev.yml up\nIf you have already ran that, just make sure that services are starting properly, and sit back.\nWaiting for $1 to start...\033[0m"
```
Comment (raquelmsmith) [inline bin/e2e-test-runner:96]:
```suggestion
  echo -e "\033[0;31m$1 isn't ready. You can run the stack with:\ndocker compose -f docker-compose.dev.yml up\nIf you've already run that, just make sure that services are starting properly and wait a bit.\nWaiting for $1 to start...\033[0m"
```

**Review by raquelmsmith — APPROVED** [review]
I wish there was a way to actually test SSO login...

File: bin/e2e-test-runner (hunk 1 above)
Comment (Twixes) [reply bin/e2e-test-runner]: Ah, this was actually more of a debugging change (copied from `bin/start`), so gonna just revert this one

**Comment by Twixes** [comment]
Based on this: https://python-social-auth.readthedocs.io/en/latest/tests.html, we _can_ do it with some test OAuth app, but it'd definitely be a bit messy with credentials and whatnot.

**Pull request merged**